packages/rpc/README.md

```diff
@@ -48,13 +48,13 @@ This is useful in cases where you need to make assertions about the suitability
 ```ts
 async function getSpecialAccountInfo(
     address: Address<'ReAL1111111111111111111111111111'>,
-    rpc: RpcMainnet,
+    rpc: RpcMainnet<unknown>,
```

Comment:
Would it not be best to use a generic type `T` here?

Reply:
`unknown` matches any more specific type, and a type parameter `T` would go unused in the return type, so it would sort of all be for naught.